pages/index.md

```diff
@@ -3,9 +3,11 @@ title: NRAU-Baltic
 titleTemplate: false
 ---
 
-<!-- script setup lang="ts">
+# NRAU-Baltic Contest 2025 is coming!
```

Comment:
Keliom eilutem zemiau yra
```
# NRAU-Baltic Contest
```

Reiktu palikt viena is ju, nes `#` sukuria `<h1>`, ju turetu but tik vienas puslapyje del SEO